src/ActivitiesSDKAdapter.js

```diff
@@ -20,6 +22,37 @@ import logger from './logger';
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/ActivitiesAdapter.js#L6}
  */
 
+/**
+ * Extracts JSON card and files from server activity.
+ *
+ * @private
+ * @param {object} sdkActivity  SDK activity object
+ * @returns {Array} Array of attachments.
+ */
+function parseSDKAttachments(sdkActivity) {
+  const attachments = [];
+
+  try {
+    if (sdkActivity.attachments) {
+      attachments.concat(sdkActivity.attachments);
+    }
+
+    if (sdkActivity.object && sdkActivity.object.cards) {
+      const card = JSON.parse(sdkActivity.object.cards[0]);
+
+      attachments.push(card);
+    }
+
+    // If object.files, then merge files.items into attachments.
+
+    return attachments;
+  } catch (err) {
+    logger.error('ACTIVITY', undefined, 'parseSDKAttachments()', `Unable parse attachments for activity with id "${sdkActivity.id}"`, err);
+
+    return [];
```

Comment:
Hi, we have a single-return policy, maybe you could try-catch just around the json parse and return attachments at the end of the function.

Reply:
Single return policy. ok will do